src/wp-includes/class-wp-duotone.php

```diff
@@ -948,45 +944,45 @@ public static function register_duotone_support( $block_type ) {
 	 * This handles selectors defined in `color.__experimentalDuotone` support
 	 * if `filter.duotone` support is not defined.
 	 *
-	 * @param string $block_name The block name.
+	 * @param WP_Block_Type $block_type Block type to check for support.
```

Comment:
Nit: according the PHPCS standards, `@param` should be right before `@return`.

So `@internal`, `@since`, and then param + return

Reply:
Plus the `@return` type is incorrect too